readthedocs/invitations/backends.py

```diff
@@ -57,7 +57,14 @@ def send_invitation(self):
         from_name = from_user.get_full_name() or from_user.username
         object_description = self._get_email_object_description()
         expiration_date = self.invitation.expiration_date
-        log.info("Emailing invitation", email=email, invitation_pk=self.invitation.pk)
+        log.info(
+            "Emailing invitation",
+            email=email,
+            invitation_pk=self.invitation.pk,
+            object_type=self.invitation.object_type,
+            object_name=self.invitation.object_name,
+            object_pk=self.invitation.object.pk,
+        )
```

Comment:
Is this showing the organization's slug or team's name? Isn't it possible to log `organization=My Organization, team=My Team`? Otherwise, it's hard to follow that relation by looking at ids.

Reply:
This will be logged as `{organization slug} {team slug}`.

>  Otherwise, it's hard to follow that relation by looking at ids.

We are already logging the type of the object, which should provide enough context for the log. So I'm not sure if there is need to generate the kwargs of the log dynamically.

Reply:
Can't we log them as keyword args? That way we can easily filter things like "all the invitation for this <organization>", which will be pretty useful while debugging.

Reply:
@humitos we should be able to do that already, just with another filter

- `object_type:organization object_name:myorg`
- `object_type:team object_name:myorg team`
- `object_type:project object_name:myproject`

It's possible to set each type as a kwargs, but then we will be generating those dynamically, which will require something like calling a method over the model to get them, or an if statement each time we create a log entry.

Reply:
Gotcha 💯 ! Thanks for the explanation!